problems/low-bus-factor.md

```diff
@@ -0,0 +1,88 @@
+---
+slug: low-bus-factor
+stages:
+  - development
+short_description: Low bus factor is a risk for team work when one or more important team members get stuck (from various reasons) and cannot work. 
+tags:
+  - Low bus factor
+  - team
+  - redundancy
+  - fresh documentation
+  - automation
+keywords:
+  - bus
```

Comment:
```suggestion
  - bus factor
```
I am afraid that the bus could seem too wide. Yes, the contextuality matters and it would probably be fine.